package.json

```diff
@@ -18,6 +18,7 @@
     "react-redux": "^7.2.6",
     "react-router-dom": "^6.0.2",
     "react-scripts": "4.0.3",
+    "redux": "^4.2.0",
```

Comment:
Its recommended using React-Toolkit in the official documentation

Reply:
https://github.com/ita-social-projects/SpaceToStudy-Client/pull/63